packages/algob/src/internal/deployer.ts

```diff
@@ -102,8 +102,8 @@ class DeployerBasicMode {
    * @param scParams parameters
    * @returns {LogicSig} loaded logic signature from assets/<file_name>.teal
    */
-  async loadLogic (name: string, scParams: LogicSigArgs): Promise<LogicSig> {
-    return await getLsig(name, scParams, this.algoOp.algodClient);
+  async loadLogic (name: string, scParams: LogicSigArgs, scInitParam?: Object): Promise<LogicSig> {
```

Comment:
Instead of using an `Object`, let's use `unknown` because we don't inherit any properties from the common JS objects.

This is a good reading: https://medium.com/@KevinBGreene/a-little-theory-with-your-typescript-top-and-bottom-types-61b380f227d

Reply:
OK